model/gamut_compress.ctl

```diff
@@ -0,0 +1,135 @@
+
+// <ACEStransformID>urn:ampas:aces:transformId:v1.5:LMT.Academy.GamutCompress.a1.0.1</ACEStransformID>
+// <ACESuserName>ACES 1.3 Look - Gamut Compress</ACESuserName>
+
+//
+// "LMT" for compressing out of gamut scene-referred values into AP1 
+// 
+// Input and output are ACES2065-1
+//
+
+//
+// Direction:
+//    By default this transform operates in the forward direction (compressing the gamut).
+//    If instead an inverse operation is needed (undoing a prior gamut compression), there
+//    is a runtime flag to do this. In ctlrender, this can be achieved by appending 
+//    '-param1 invert 1' after the '-ctl gamut_compress.ctl' string.
+//
+
+
+
+import "ACESlib.Transform_Common";
+
+
+
+/* --- Gamut Compress Parameters --- */
+const float threshold_c = 0.815;
+const float threshold_m = 0.803;
+const float threshold_y = 0.880;
+const float power = 1.2;
+const float cyan =  0.147;
+const float magenta = 0.264;
+const float yellow = 0.312;
+
+
+
+// calculate compressed distance
+float compress(float dist, float lim, float thr, bool invert, float power)
+{
+    float cdist;
+    float s;
+    if (dist < thr) {
+        cdist = dist;
+    }
+    else { // power(p) compression function plot https://www.desmos.com/calculator/54aytu7hek
```

Comment:
this should probably be removed
not sure how long desmos keeps these links active

Reply:
Agreed, we can host these links in the separate documentation instead.